case-studies/eshoppen.md

```diff
@@ -153,6 +153,50 @@ For this component:
 * RR: A virtual machine with 2 vCPUs was used, this data was sourced from [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 * TR: The bare metal host is split up into 16 vCPUs in total. This data was sourced from the [Cloud Carbon Footprint Azure Instances Coefficients](https://github.com/cloud-carbon-footprint/cloud-carbon-coefficients/blob/main/data/azure-instances.csv).
 
+
+### Front End Browser for Web application
```

Comment:
Correct me if I'm wrong but the model here is assuming that all the requests will come from one browser on one device and essentially be making 50 simultaneous requests over a 1hr period. So like 500 people all sharing one laptop over 1hr.

I think the right solution here is to model the reality which is multiple users across multiple devices hitting a centralised service. That's more accurate and will be the kind of model people will copy and use in other case studies.

Reply:
So model the project for one User for checking out a single basket and then measure SCI for that one user and multiply that one number by 500